src/frontend/src/app/pages/profile-edit/admin-personal-info-form/admin-personal-info-form.component.ts

```diff
@@ -102,26 +100,26 @@ export interface AdminProfileForm {
       </app-key-value-grid>
 
       <div class="flex items-center justify-end">
+        @if (profileForm.invalid) {
+          <div class="text-error pr-5 text-sm md:pr-10">
+            Fix the validation errors
+          </div>
+        }
         <button class="btn btn-outline mr-4" (click)="cancelEdits()">
           Cancel
         </button>
 
         <button
-          #submitButton
           type="submit"
-          [appTooltip]="
-            profileForm.invalid
-              ? 'Fix the validation errors'
-              : isSaving
-                ? 'Saving...'
-                : null
-          "
+          [attr.aria-label]="isSaving ? 'Saving' : 'Save'"
           [disabled]="profileForm.invalid || isSaving"
-          class="btn"
+          class="btn relative"
           [ngClass]="isSaving ? 'text-transparent' : ''"
```

Comment:
Use `aria-hidden` so that this text is not visible to screen readers. Move this and `text-transparent` onto an element wrapping the `Save` text